contrib/managed-azure/setup_dev_environment.md

```diff
@@ -0,0 +1,279 @@
+# Setting Up a Development Environment
+
+## Prerequisites
+
+- Azure CLI
+- jq
+- oc (OpenShift CLI)
+- kubectl
+
+## Variables
+
+Set the following variables according to your environment:
+
+```sh
+PERSONAL_SP_NAME=<NAME_OF_PERSONAL_SP>
+LOCATION="eastus" 
+PULL_SECRET=<PATH_TO_PULLSECRET>
+PERSISTENT_RG_NAME="os4-common"
+SP_AKS_CREDS=<PATH_TO_AKS_CREDS>
+RELEASE_IMAGE=<OCP_PAYLOAD_RELEASE_IMAGE>
+AKS_CP_MI_NAME=<MANAGED_IDENTITY_NAME_FOR_AKS_CLUSTER> 
+AKS_KUBELET_MI_NAME=<KUBELET_MANAGED_IDENTITY_NAME_FOR_AKS_CLUSTER>
+KV_NAME=<KV_NAME>
+AZURE_DISK_SP_NAME="azure-disk-<PERSONAL_SP_NAME>"
+AZURE_FILE_SP_NAME="azure-file-<PERSONAL_SP_NAME>"
+NODEPOOL_MGMT="nodepool-mgmt-<PERSONAL_SP_NAME>"
+CLOUD_PROVIDER_SP_NAME="cloud-provider-<PERSONAL_SP_NAME>"
+CNCC_NAME="cncc-<PERSONAL_SP_NAME>"
+CONTROL_PLANE_SP_NAME="cpo-<PERSONAL_SP_NAME>"
+IMAGE_REGISTRY_SP_NAME="ciro-<PERSONAL_SP_NAME>"
+INGRESS_SP_NAME="ingress-<PERSONAL_SP_NAME>"
+USER_ACCOUNT_ID=<ID_OF_USER_ACCOUNT_OR_SP>
+SP_OUTPUT_FILE=<OUTPUT_FILE_FOR_CREATED_SPs>
+AKS_CLUSTER_RG_NAME=<AKS_CLUSTER_RG_NAME>
+DNS_RECORD_NAME=<DNS_RECORD_NAME>
+EXTERNAL_DNS_SP_NAME=<EXTERNAL_DNS_SP_NAME>
+EXTERNAL_DNS_CREDS=<PATH_TO_FILE_WITH_DNS_CREDS>
+DNS_ZONE_NAME="<DNS_RECORD_NAME>.hypershift.azure.devcluster.openshift.com"
```

Comment:
Should this block and the two below be generic and not point to our actual resources?